src/services/PassService.js

```diff
@@ -33,6 +33,28 @@ const handlePass = async (passId, user) => {
   await conversationConfig.create();
 }
 
+/**
+ * Create a pass
+ * 
+ * Verifies if the user already has a pass, otherwise creates it
+ *
+ * @param {Parse.User} user
+ */
+ const createPass = async user => {
+  try {
+    const currentPass = await new Parse.Query('Pass').equalTo('owner', user).first();
+    if (currentPass) return currentPass;
+    const pass = new Parse.Object('Pass');
+    pass.set('link', 'ADD_LINK');
```

Comment:
@Benjamindodgson I saw all our current Pass have this set to "ADD_LINK", not sure what that means, but just in case I set it to match the others